projects/aws/upgrader/docker/linux/Dockerfile

```diff
@@ -22,6 +22,7 @@ COPY _output/$RELEASE_BRANCH/dependencies/$TARGETOS-$TARGETARCH/eksd/kubernetes/
     _output/$RELEASE_BRANCH/dependencies/$TARGETOS-$TARGETARCH/eksd/kubernetes/server/bin/kubeadm \
     /eksa-upgrades/binaries/kubernetes/usr/bin/
 COPY ../../upgrade.sh /eksa-upgrades/scripts/
```

Comment:
do we still need the sh script?

Reply:
No we don't. but i was planning to remove the script in a separate PR once the eks-a [change](https://github.com/aws/eks-anywhere/pull/7927) for consuming the binary merges not to break CI. Given the nightly runs we shouldn't have problem but still ideally we want to remove once that is merged.